python/tests/test_topology.py

```diff
@@ -232,45 +298,49 @@ def test_same_tree_zero_distance(self):
         for n in range(2, 10):
             for seed in range(1, 10):
                 ts = msprime.simulate(n, random_seed=seed)
-                tree = ts.first()
+                tree = next(ts.trees(sample_lists=True))
```

Comment:
I couldn’t find a place in Python to provide the sample_lists flag other than the trees iterator. Is there a better way to do this?

Reply:
This is one way to do it. The other is 
```python
tree = tskit.Tree(ts, sample_lists=True)
tree.first()
```
I guess we should add kwargs to the ``first()`` etc methods.

Reply:
The way you have is better than using the low-level API though.